.travis.yml

```diff
@@ -1,34 +1,42 @@
-sudo: false
+dist: trusty
+sudo: required
 language: go
 
 go:
   - 1.7.x
   - 1.8.x
+  - master
 
 services:
-  - mysql
-  - mongodb
-
-notifications:
-  email: false
+  - docker
 
 install:
   - go get -u github.com/Masterminds/glide
   - rm -rf vendor/
   - glide install --strip-vendor
 
+before_script:
+  - sudo service mysql stop
```

Comment:
Wow, is that really required? Since `mysql` is not in services it shouldn't be available. I remember I needed (in some other project) to add `services: mysql` just to get `mysql` client binary... so weird.